fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/utility/handler/RollCallHandlerTest.java

```diff
@@ -148,6 +147,9 @@ public void testHandleCreateRollCall()
   public void testHandleOpenRollCall()
       throws DataHandlingException, UnknownLaoException, UnknownRollCallException,
           UnknownElectionException, NoRollCallException {
+    // Check if the Roll Call can be opened
+    assertTrue(rollCallRepo.canOpenRollCall(LAO.getId()));
```

Comment:
It might be better to create a dedicated test for the blocking functionality instead of inserting it into both testHandleOpenRollCall() and testHandleCloseRollCall()

Reply:
Yes, you're right, it's definetely cleaner like that